linode/manager
# hotfix: Fix Core Plan table display

## Description 📝
Fix edge/distributed plans displaying when a core region is selected. This was because the class changed to `dedicated` and we missed logic to also handle filtering out `dedicated-edge` from the plan table so everything dedicated was displaying.

Testing will be addressed in M3-8234

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/115299789/418bdf85-eae6-4f26-9de0-2ed5cb588a1f) | ![image](https://github.com/linode/manager/assets/115299789/fe8c00ed-bbf4-4875-ae8e-e67b0bbaf64c) |

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Point to the dev environment and ensure your account has the `new-dc-testing`, `edge_testing` and `edge_compute` customer tags

### Reproduction steps
(How to reproduce the issue, if applicable)
- On remote dev, go to the Linode Create flow and select a Core region. The Dedicated CPU tab shows additional, more expensive dedicated plans
- Click on the Shared CPU tab. Observe a $6 Nanode at the bottom

### Verification steps
(How to verify changes)
- Check this branch out locally pointing to dev. Repeat reproduction steps and the duplicated more expensive edge/distributed plans should no longer be displaying
- Selecting an edge/distributed region should show it's own table with no tabs

## As an Author I have considered 🤔

*Check all that apply*
